Add plugin: Image Tools

Hosstell opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

https://github.com/Hosstell/image-tools-obsidian-plugin
Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["fundingUrl": "https://github.com/Hosstell",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/manifest.json#L9)
This is meant for links to services like _Buy me a coffee_, _GitHub sponsors_ and so on, if you don't have such a link remove this line.

["https://github.com/Hosstell/image-editor-obsidian-plugin/blob/main/static/align-left.png?raw=true",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/src/align-icons-item.ts#L36), ["https://github.com/Hosstell/image-editor-obsidian-plugin/blob/main/static/align-center.png?raw=true",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/src/align-icons-item.ts#L45), ["https://github.com/Hosstell/image-editor-obsidian-plugin/blob/main/static/align-right.png?raw=true",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/src/align-icons-item.ts#L54), ["https://github.com/Hosstell/image-editor-obsidian-plugin/blob/main/static/copy.png?raw=true",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/src/copy-image-item.ts#L24), ["https://github.com/Hosstell/image-editor-obsidian-plugin/blob/main/static/zoom.png?raw=true",](https://github.com/Hosstell/image-tools-obsidian-plugin/blob/c6b1d637e0d85a30c31b8b61639dc81f23c6edb2/src/zoom-icon-item.ts#L25)
These images need to be bundled into the plugin, not loaded from an external server.
Even better would be if you used icons from the Lucide library, as that is bundled with Obsidian and you can just do: `setIcon(element, 'icon')`

Hosstell commented:
@joethei 

1. Removed '"fundingUrl": "https://github.com/Hosstell"' from manifest.yml
2. Replaced icons urls to Lucide library. Icons files are removed from repo.

Check it please again =)
